src/common/config.h

```diff
@@ -9,13 +10,29 @@ namespace raftcpp {
 
 class Config final {
     public:
-    static Config From(const std::string &config_str) { return Config(); }
```

Comment:
The ctor of Config should be private or deleted.

Reply:
Done